utilities/isDefaultRole.js

```diff
@@ -1,7 +1,12 @@
 import Group from "../classes/Group/Group.js"
 
 export default function scrubDefaultRoles(roleName) {
-  const defaultRoles = Object.keys(Group.defaultRoles)
+  const defaultRoles = Object.keys({
```

Comment:
I don't love this moving here instead of being defined in Group. Is there a good reason?